test/cmd/images.sh

```diff
@@ -72,7 +72,7 @@ os::cmd::expect_success_and_text 'oc describe istag/mysql:latest' 'Image Created
 os::cmd::expect_success_and_text 'oc describe istag/mysql:latest' 'Image Name:'
 name=$(oc get istag/mysql:latest --template='{{ .image.metadata.name }}')
 imagename="isimage/mysql@${name:0:15}"
-os::cmd::expect_success 'oc describe "${imagename}"'
+os::cmd::expect_success "oc describe ${imagename}"
```

Comment:
*Ensuring* that your variables don't get expanded before they get passed in might be very hard - if you are trying to pass in a string that contains single-quotes, it's going to look ugly and not be very clear if we also have to use single-quotes around the entire thing to make sure nothing gets expanded.

The added benefit of seeing the actual command being run is pretty useful, too.